src/common/login.js

```diff
@@ -14,7 +14,7 @@ const Login = (() => {
 
     const loginUrl = () => {
         const server_url = localStorage.getItem('config.server_url')
-        const language = localStorage.getItem('i18n')
+        const language = localStorage.getItem('i18n').replace('-', '_')
```

Comment:
```suggestion
        const language = localStorage.getItem('i18n')?.replace('-', '_')
```

can have optional chaining feature